sdk/core/core/src/az_http_policy_retry.c

```diff
@@ -107,7 +107,7 @@ AZ_INLINE AZ_NODISCARD az_result _az_http_policy_retry_get_retry_after(
     {
       if (az_span_is_content_equal_ignoring_case(header.key, AZ_SPAN_FROM_STR("retry-after-ms"))
           || az_span_is_content_equal_ignoring_case(
-              header.key, AZ_SPAN_FROM_STR("x-ms-retry-after-ms")))
+                 header.key, AZ_SPAN_FROM_STR("x-ms-retry-after-ms")))
```

Comment:
is this space coming from Ctrl+K+D (auto-format)?

Reply:
yes Sr.